src/os/bluestore/BlueStore.cc

```diff
@@ -9857,8 +9857,8 @@ int BlueStore::queue_transactions(
     c->complete(0);
   }
   if (on_applied.size()) {
-    if (txc->ch->commit_queue)
```

Comment:
please squash this change into 4ca3ccb21cb89007e54ae760c055ceee912a9a0f . in general, we don't fix a fix in the same PR.